<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analyse_Binary_Img</class>
 <widget class="QDialog" name="Analyse_Binary_Img">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyse Binary Image</string>
  </property>
  <widget class="QDialogButtonBox" name="btn_ok_cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OpenImage">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>106</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_img_list">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_GTI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Binary Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>← &amp;Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&amp;Next →</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtBrw_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>241</width>
     <height>57</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>128</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="rdbtn_single">
      <property name="text">
       <string>single</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdbtn_batch">
      <property name="text">
       <string>batch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>461</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="chkB_BlackPix_Coverage">
      <property name="text">
       <string>Black Pixel → Coverage   |</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_file_name_csv">
      <property name="text">
       <string>Dataset Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputtxt_file_name_csv">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_ok_cancel</sender>
   <signal>accepted()</signal>
   <receiver>Analyse_Binary_Img</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ok_cancel</sender>
   <signal>rejected()</signal>
   <receiver>Analyse_Binary_Img</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
